integrationtest/neo/node/Node.d

```diff
@@ -64,6 +64,7 @@ public class Node : NodeBase!(ConnHandler)
 
         Options options;
         options.epoll = epoll;
+        options.no_delay = true;
```

Comment:
Is no-delay already set on the client side?

Reply:
It is: https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/integrationtest/neo/client/Client.d#L635 (`enableSocketNoDelay()`)